lib/rbs/collection/config/lockfile_generator.rb

```diff
@@ -161,11 +161,6 @@ def generate
           return if lockfile.gems.key?(name)
 
           case name
-          when 'rubygems', 'set'
-            msg = "`#{name}` has been moved to core library, so it is always loaded. Remove explicit loading `#{name}`"
-            msg << " from `#{from_gem}`" if from_gem
-            msg << "."
-            return
           when *ALUMNI_STDLIBS.keys
```

Comment:
```suggestion
          if ALUMNI_STDLIBS.key? name
```
(:

Reply:
Thanks for the review.
I want to keep the `case` statement as I plan to add another branch later.